### ISSUE TITLE
Use better icon for the "Change language" button

### ISSUE THREAD

**Issue by Vanadiae** [hugolabe/Wike#27]
At first I didn't quite understand what this button did. It's also used in the preferences window's view switcher. But now I've understood it's to change the language of the page. The little flag icon doesn't ring a bell here on what it's about, so maybe replacing it with `globe-symbolic` would make it clearer, as that's generally the kind of icon websites use for that purpose (and anyway seeing a globe like that seems nearer to a "Language" button than a flag…).

**Comment by hugolabe** [comment]
I tried both icons at the time, but I like the flag option better.

At the level of clarity I don't see a big difference that justifies the change (eg GNOME Settings uses a flag icon in the Region & Language section). As in issue #25 I think the tooltip addiction would better solve any confusion about the function of the button.

**Comment by Vanadiae** [comment]
Tooltips are only meant to show extra informations, not to communicate the whole purpose of the button. The [GNOME HIG](https://developer.gnome.org/hig/patterns/feedback/tooltips.html#when-to-use) has more explanations.

**Comment by apandada1** [comment]
> Tooltips can be set for any UI element. They are primarily used to convey **additional information about what controls (often icon buttons) do**, but can also be used to show additional information about application content.

_(emphasis mine)_

It will show the additional information that the button changes the language. 
Even if the icon were changed to `globe-symbolic`, one may not understand that it is for changing language. We would need a tooltip to explain that.

**Comment by Vanadiae** [comment]
That's what it is: additional information. It shouldn't be the primary way of conveying the purpose of the button. Tooltips and proper icon don't have to exclude each other, both should be used. Of course some might not understand that the globe indicates it's to change the language, but it's IMO still clearer than the flag icon currently used.

**Comment by hugolabe** [comment]
I have added tooltips to some header buttons in a recent commit. I think they are necessary and almost every application I know uses them to indicate what a control does.

It is true that the icon should be the first source of information for the user. But here I think it is a question of opinions. I have seen in many places both metaphors used to refer to a change in language or location.

In my opinion changing the icon does not improve anything, so I prefer to stick with the option I chose from the beginning ;-)
